Blaise.Tests.Helpers/Configuration/BlaiseConfigurationHelper.cs

```diff
@@ -10,12 +10,13 @@ public static class BlaiseConfigurationHelper
         public static string InstrumentPath => ConfigurationExtensions.GetVariable("InstrumentPath");
         public static string InstrumentName => ConfigurationExtensions.GetVariable("InstrumentName");
         public static string SecondInstrumentName => ConfigurationExtensions.GetVariable("SecondInstrumentName");
-        public static string InstrumentPackage => $"{InstrumentPath}//{InstrumentName}.bpkg";
+        public static string InstrumentPackage => $@"{InstrumentPath}\{InstrumentName}.bpkg";
         public static string SecondInstrumentPackage => $"{InstrumentPath}//{SecondInstrumentName}.bpkg";
```

Comment:
The path separator has been changes on the line above, but not on this line - it should probably be consistent.

Reply:
Changed this